src/components/atoms/Input/Component.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ReactComponent as Delete } from './Icons/delete.svg';
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledLabel = styled.label`
+  color: #828282;
+  font-weight: 400;
+  font-size: 14px;
+  text-align: left;
+  margin-bottom: 11px;
+`;
+
+const InputContainer = styled.div<Props>`
+  display: flex;
+  background: white;
+  width: 260px;
+  height: 32px;
+  border: solid 1px #828282;
+  border-radius: 4px;
+  align-items: center;
+  justify-content: space-between;
+
+  &:focus-within {
+    outline: 2px solid #2f80ed;
+  }
+`;
+
+const TextInput = styled.input`
+  border: none;
+  border-radius: 4px;
+  font-size: 14px;
+  width: 100%;
+  margin-inline: 12px;
+
+  &:focus {
+    outline: none;
+  }
+`;
+
+const IconWrapper = styled.button`
+  display: flex;
+  width: 20px;
+  height: 20px;
+  align-items: center;
+  background-color: #828282;
+  border: 0;
+  border-radius: 50%;
+  cursor: pointer;
+  margin-right: 6px;
+`;
+
+export type Props = {
+  label?: string;
+  inputType?: string;
```

Comment:
Instead of just string, can you make this more specific? Because if it's just string I can put anything like "test" and it will be accepted by typescript but might cause an error during runtime since "test" is not a valid type for input.

Maybe you can set it to `type: 'text' | 'password'`. What do you think?

Reply:
I initially set it to ``type: 'text' | 'password'``, but I figured future developers might use it for other input types like email. How about ``type: 'text' | 'email' | 'password'``

Reply:
@jhonangelo I think that's okay

Reply:
Thank you; I'll make the necessary changes right away.